### PR TITLE
Fixes #85

### LINKED ISSUE #85: Rewrite openHAB info activity into a fragment
To simplify navigation and future development of the app openHAB information activity should be rewritten into a fragment. Dependant on #83

### LINKED ISSUE #155: Error when try to compile
i have error when want to compile it source code, help please
error openhab

### PR DESCRIPTION
**Pull request by CristianSarac**
Fixes #85 
Also contains this Fix #155

### CODE REVIEW

**Comment by slawekjaranowski** [comment]
Hi,
To support multiDex in older version of android please look into #147, #158

**Comment by CristianSarac** [comment]
Thanks @slawekjaranowski  
It should be fixed now

**Comment by digitaldan** [comment]
Hi @CristianSarac , the multi dex fix #158 has been merged, can you resolve the conflicts and submit again?  Thanks!

**Comment by CristianSarac** [comment]
Yeah sure i will do that

**Comment by CristianSarac** [comment]
I fixed the conflicts and updated the pull request

**Comment by digitaldan** [comment]
Thanks @CristianSarac  I will review shortly.

File: mobile/src/main/java/org/openhab/habdroid/ui/OpenHABInfoFragment.java [hunk 1]
```diff
@@ -39,28 +43,49 @@
...
+            mPassword = bundle.getString("password");
             mAsyncHttpClient.setBasicAuth(mUsername, mPassword);
-        } else {
+        }else {
             Log.e(TAG, "No openHABBaseURl parameter passed, can't fetch openHAB info from nowhere");
-            finish();
+
+        }
+
+        //getDialog().setTitle("OpenHabInfo");
```
Comment (digitaldan) [inline mobile/src/main/java/org/openhab/habdroid/ui/OpenHABInfoFragment.java]: Please remove if not needed.

**Comment by digitaldan** [comment]
Hi I am testing this now, so far so good.  I had one minor comment on the code.  Can you resolve the conflicts in this branch and then squash the PR into a single commit?  Thanks!

**Comment by digitaldan** [comment]
Hi @CristianSarac , I think something has gone wrong with the latest commit ;-)  can you try fixing the conflicts and squash everything into a single commit?

File: mobile/src/main/java/org/openhab/habdroid/ui/OpenHABInfoFragment.java [hunk 2]
```diff
@@ -35,28 +39,48 @@
...
+        View view = inflater.inflate(R.layout.openhabinfo, container);
+        mAsyncHttpClient = new MyAsyncHttpClient(getActivity().getApplicationContext());
+        mOpenHABVersionText = (TextView)view.findViewById(R.id.openhab_version);
+        mOpenHABUUIDText = (TextView)view.findViewById(R.id.openhab_uuid);
+        mOpenHABSecretText = (TextView)view.findViewById(R.id.openhab_secret);
+        mOpenHABSecretLabel = (TextView)view.findViewById(R.id.openhab_secret_label);
+        Bundle bundle=getArguments();
+
+        if (bundle!=null)
+        {
```
Comment (digitaldan) [inline mobile/src/main/java/org/openhab/habdroid/ui/OpenHABInfoFragment.java]: I know this is a bit of a nick-pick, but can you format you code to match the current style, with the '{' being on the same line as the statement?

File: mobile/src/main/java/org/openhab/habdroid/ui/OpenHABInfoFragment.java [hunk 3]
```diff
@@ -35,28 +39,48 @@
...
+
+            mOpenHABBaseUrl = bundle.getString("openHABBaseUrl");
+            mUsername = bundle.getString("username");
+            mPassword = bundle.getString("password");
             mAsyncHttpClient.setBasicAuth(mUsername, mPassword);
-        } else {
+        }else {
```
Comment (digitaldan) [inline mobile/src/main/java/org/openhab/habdroid/ui/OpenHABInfoFragment.java]: Please insert a space like " } else {'

**Comment by CristianSarac** [comment]
I fixed the conflicts. When i try to rebase the branch i get all sorts of errors, if you want i can make another PR with just one commit or you can squash them using the merge button [check this link ](http://stackoverflow.com/questions/11396069/squash-to-only-one-proper-commit-for-github-pull-request/36377439#36377439)

**Comment by digitaldan** [comment]
Just wanted to comment, I have not forgot about this ;-) I have a few other things on my plate this week, but my plan is to test and merge this by the weekend.  After I test I will try the squash feature.  FYI i have had success doing my squashes this way when on a feature branch:
```
git reset origin/master
git add ./
git commit -s
```

**Pull request merged**